packages/buildcop/test/buildcop.ts

```diff
@@ -96,20 +100,11 @@ describe('buildcop', () => {
   let probot: Probot;
 
   beforeEach(() => {
-    probot = new Probot({
-      // use a bare instance of octokit, the default version
-      // enables retries which makes testing difficult.
-      // eslint-disable-next-line node/no-extraneous-require
-      Octokit: require('@octokit/rest').Octokit,
+    probot = createProbot({
+      githubToken: 'abc123',
+      Octokit: TestingOctokit as any,
```

Comment:
A bunch of `any` lint warnings in this file, too.

Reply:
Yeah, unfortunately the 'any' is a known issue since the Github API event names have been locked down; see here: https://github.com/octokit/webhooks.js/issues/277

Reply:
I'll add this as a comment in the file, though!